src/main/scala/com/cognite/sdk/scala/v1/sequenceRows.scala

```diff
@@ -12,21 +13,41 @@ final case class SequenceRowsInsertByExternalId(
 )
 final case class SequenceRowsDeleteById(id: Long, rows: Seq[Long])
 final case class SequenceRowsDeleteByExternalId(externalId: String, rows: Seq[Long])
+sealed trait SequenceRowsQuery {
+  val start: Option[Long]
+  val end: Option[Long]
+  val limit: Option[Int]
+  val cursor: Option[String]
+  val columns: Option[Seq[String]]
+
+  def withCursorAndLimit(newCursor: Option[String], limit: Option[Int]): SequenceRowsQuery
+}
 final case class SequenceRowsQueryById(
     id: Long,
-    start: Long,
-    end: Long,
+    start: Option[Long],
```

Comment:
We could also do `start: Long = 0L`, `end: Long = Long.MaxValue` (or similar, if it should be 2^53 or some such instead for the max value), as we do with for example
```scala
final case class AssetsQuery(
    filter: Option[AssetsFilter] = None,
    search: Option[AssetsSearch] = None,
    limit: Int = 100
) extends SearchQuery[AssetsFilter, AssetsSearch]
```

Reply:
Hmm, I'm not sure if I like it more. Esp, for the upper limit, they don't really say in the docs what's the upper bound.

Reply:
The key is a long, `integer <int64> >= 1`, so it certainly has an upper bound.